### ISSUE TITLE
DUC_HDC performance

### ISSUE THREAD

**Issue by InstantWindy** [saeedizadi/binseg_pytoch#1]
Hello! Do you use the 'DUC_HDC‘’ model train the Voc2012 dataset? I use it ,but I found the result was not good, it appears the griding situation,But it should not be the case, because the model is the solution to the grid phenomenon.I don't why ,could you help me? Thanks!

**Comment by saeedizadi** [comment]
Hi, 

Yes, I'm getting the grid-like pattern as well. The implementation is correct, and I'm skeptical to the theory behind the DUCHDC method.

**Comment by InstantWindy** [comment]
Thanks! I have another question. If it is a binary classification,  using BCELoss to calculate the loss, how do you calculate the accuracy and mean IoU? Is BCELoss good for image segmentation? I'm sorry to trouble you

**Comment by saeedizadi** [comment]
BCE stands for binary cross entropy. So, it's suitable for binary segmentation task. The accuracy and IoU can be simply calculated for binary segmentation tasks. Just treat each pixel as a sample. Yes, BCE is good for binary segmentation.
